<launch>
    <node pkg="voxel_plus" type="lio_node" name="lio_node" output="screen">
        <rosparam command="load" file="$(find voxel_plus)/config/mapping.yaml" />
    </node>

    <!-- <rosparam command="load" file="$(find std_matcher)/config/raw.yaml" /> -->
    
    <node pkg="std_matcher" type="online_demo" name="online_demo" output="screen">
        <!-- <rosparam command="load" file="$(find std_matcher)/config/config.yaml" /> -->
    </node>
    <!-- <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz"  args="-d $(find voxel_plus)/rviz/std.rviz"/> -->

</launch>